components/ButtonLink/ButtonLink.module.css

```
@@ -0,0 +1,10 @@
+.link {
+  display: inline-block;
+  color: white;
+  border-radius: 35px;
+  background: black;
+  font-size: 1.2em;
+  padding: 0.5em 1em;
+  text-align: center;
+  border: white 10px solid;
```

i would recommend em or rem instead of px 
```suggestion
  border: white 0.6em solid;
```